src/js/transforms.js

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
+*/
+
+/* global fluid, sjrk */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("sjrk.storyTelling.transforms");
+
+    fluid.defaults("sjrk.storyTelling.transforms.stringToArray", {
```

Comment:
Is it worth contributing these to Infusion? They seem generally useful.

Reply:
I think they're quite useful, and as you've suggested outside of this discussion, there doesn't seem to be an existing equivalent in Infusion already.

The closest I was able to find in the existing transforms is [`fluid.transforms.JSONstringToObject`](https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/ModelTransformationTransforms.js#L854-L857) and its inverse function, [`fluid.transforms.objectToJSONString`](https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/ModelTransformationTransforms.js#L854-L857), but these deal with JSON objects or serializations, and in this case we'd like to have it produce arrays, specifically.

A follow-up question: is it worth perhaps using the above-mentioned transform instead of the one we've implemented in `transforms.js`?